libs/sql-schema-describer/src/postgres.rs

```diff
@@ -214,16 +205,64 @@ impl SqlSchemaDescriber {
     }
 
     fn get_precision(col: &ResultRow) -> Precision {
-        let character_maximum_length = col.get_u32("character_maximum_length");
-        let numeric_precision = col.get_u32("numeric_precision");
-        let numeric_precision_radix = col.get_u32("numeric_precision_radix");
-        let numeric_scale = col.get_u32("numeric_scale");
-        let time_precision = col.get_u32("datetime_precision");
+        let (character_maximum_length, numeric_precision, numeric_scale, time_precision) =
+            if matches!(col.get_expect_string("data_type").as_str(), "ARRAY") {
+                fn get_single(formatted_type: &String) -> Option<u32> {
+                    static SINGLE_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r#".*\(([0-9]*)\).*\[\]$"#).unwrap());
```

Comment:
These regexps should be in the top of the file, documented what they match.

Reply:
I'll talk to tom about establishing a uniform style in the describer.

Reply:
Just saying because regexps are never easy to read for anybody, even for the person who wrote them after a few weeks. You could maybe do examples like here? https://github.com/prisma/prisma-engines/blob/master/libs/sql-schema-describer/src/mssql.rs#L12-L27

Reply:
Yeah, that's a good idea, but I'll do it in a follow up